Formula/w/wcstools.rb

```diff
@@ -0,0 +1,19 @@
+class Wcstools < Formula
+  desc "Tools for using World Coordinate Systems (WCS) in astronomical images"
+  homepage "http://tdc-www.harvard.edu/wcstools/"
+  url "http://tdc-www.harvard.edu/software/wcstools/wcstools-3.9.7.tar.gz"
+  sha256 "525f6970eb818f822db75c1526b3122b1af078affa572dce303de37df5c7b088"
+  license "LGPL-2.0-only"
+
+  depends_on :macos
```

Comment:
This doesn't seem accurate, since the homepage has references to using this on Linux.

Reply:
yes, but I don't have a linux machine now, maybe someone can help.

Reply:
Should I close this pull or wait for someone help？

Reply:
You can just remove this line and let this build on Linux CI.

Reply:
as `Makefile.osx` only support for macos, need other file for built on linux, I've not test it on linux

Reply:
Just `make all` works on macOS (based on instructions in the `Install` file), this should work on Linux too. You can update the formula and it can be tested on Linux CI.